docs/src/use/configure/ignore.md

```diff
@@ -12,6 +12,10 @@ eleventyNavigation:
 This page explains how to ignore files using the flat config format. For the deprecated eslintrc format, [see the deprecated documentation](ignore-deprecated).
 :::
 
+::: tip
+This page explains how to use `ignores` property of an ESLint configuration object. The `ignores` property can behave as global or non-global, based on how you use it. Please read how [ignores can be used as global or non-global](configuration-files) before going through this page.
```

Comment:
```suggestion
This page explains how to use `ignores` property of an ESLint configuration object to globally ignore files and directories. The `ignores` property can behave as either global or non-global depending on how you use it. For more information on non-global ignores, see [Specifying files and ignores](configuration-files#specifying-files-and-ignores).
```

Reply:
Added in https://github.com/eslint/eslint/pull/19297/commits/45ae8059c9807d02a53f471f1bc9a47b38c63997 with an additional line.